config/environments/test.rb

```diff
@@ -12,7 +12,7 @@
   # Do not eager load code on boot. This avoids loading your whole application
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  config.eager_load = true
```

Comment:
Just wondering why. Does it improve run time?

Reply:
Just some leftover. Removed, thanks.